### PR TITLE
Support CytAssist protocol; Clear hardcoded step ids

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

File: LIMS2DB/objectsDB/process_categories.py [hunk 1]
```diff
@@ -79,6 +79,8 @@
     '2205' : 'Adapter ligation and reverse transcription',
     '2254' : 'ONT End-Prep and Cleanup',
     '2360' : 'Library Preparation & Amplification',
+    '2656' : 'Deparafinization, H&E, Tissue Imaging',
```
Comment (aanil) [inline LIMS2DB/objectsDB/process_categories.py:82]: 2656 and 2657 count as both Prepstart and Workset?

**Review by aanil — APPROVED** [review]
👍

File: LIMS2DB/objectsDB/process_categories.py (hunk 1 above)
Comment (chuan-wang) [reply LIMS2DB/objectsDB/process_categories.py:82]: Yes, this protocol was not properly set up in LIMS, and I have no other option to fix it. Later I will talk with someone to re-do the protocol.

Comment (aanil) [reply LIMS2DB/objectsDB/process_categories.py:82]: So is the PR ok to merge with this?

Comment (chuan-wang) [reply LIMS2DB/objectsDB/process_categories.py:82]: Yes, for now it is OK to merge. I will talk with Alfred to change to protocol when he is back.

**Pull request merged**